mm2src/coins/utxo/utxo_common.rs

```diff
@@ -2329,8 +2329,9 @@ where
                 Ok(value) => value,
                 Err(e) => match &e.error {
                     JsonRpcErrorType::InvalidRequest(e)
+                    | JsonRpcErrorType::Parse(_, e)
                     | JsonRpcErrorType::Transport(e)
-                    | JsonRpcErrorType::Parse(_, e) => {
+                    | JsonRpcErrorType::Internal(e) => {
```

Comment:
I think we need to discuss whether we need to retry the TX history request or not if we got an internal error.
@shamardy what do you think?

Reply:
You are right, I don't think we need to retry the TX history request if we got an internal error, but we need to be careful with using `JsonRpcErrorType::Internal` in the future as it should be used for only non-recoverable errors.